togetherly/lib/providers/chore_provider.dart

```diff
@@ -26,18 +29,26 @@ class ChoreProvider with ChangeNotifier {
   //   return personId == null ? null : choresAssignedToPerson(personId);
   // }
 
-  Future<void> addChore(Chore chore) async {
-    await _service.insertChore(chore);
+  Future<void> addChore(Chore chore, children) async {
+    chore = await choreService.insertChore(chore);
+    for (final c in children) {
+      Assignment assignment = Assignment(personId: c, choreId: chore.id!);
+      await assignmentService.insertAssignment(assignment);
+    }
     await refresh();
   }
 
-  Future<void> deleteChore(Chore chore) async {
-    await _service.deleteChore(chore);
+  Future<void> deleteChore(Chore chore, children) async {
```

Comment:
Deleting a chore should not require passing the list of children the chore is assigned to. If the foreign keys are set up properly in the database, any associated assignment rows should be deleted automatically when a chore is deleted. If the intent is to provide a way to change who is assigned to the chore, that should be done via a different method.

Reply:
Yes, I was thinking that it could be deleting that chore for multiple children or just one and keep the instance of chore created for another child. Can you add this on the agenda for next meeting?